source/MRMesh/MRConeObject.cpp

```diff
@@ -0,0 +1,181 @@
+#include "MRConeObject.h"
+
+#include "MRMatrix3.h"
+#include "MRCylinder.h"
+#include "MRMesh.h"
+#include "MRObjectFactory.h"
+#include "MRPch/MRJson.h"
+#include <Eigen/Dense>
+#include "MRConeApproximator.h"
+#include "MRMeshFwd.h"
+#include "MRLine.h"
+#include "MRGTest.h"
+
+#include <iostream>
+#include "MRMeshNormals.h"
+#include "MRMeshSubdivide.h"
+#include "MRArrow.h"
+
+namespace MR
+{
+
+namespace
+{
+constexpr int cDetailLevel = 64;
+constexpr float thicknessArrow = 0.01f;
+constexpr float cBaseRadius = 1.0f;
+constexpr float cBaseLength = 1.0f;
+
+constexpr MR::Vector3f base = MR::Vector3f::plusZ();
+constexpr MR::Vector3f apex = { 0,0,0 };
+
+
+
+float getFeatureRediusByAngle( float angle )
```

Comment:
Typo, `redius`->`radius`. Same in the next function.

Reply:
:)